### ISSUE TITLE
low fps on iphone 6 plus

### ISSUE THREAD

**Issue by xiaobaker** [pixijs/pixi.js#4206]
pixi.js is doing amazingly well for us on all our android and ios devices except iphone 6 plus. First discovered in our own game, later on we found that it can be successfully reproduced using http://www.goodboydigital.com/pixijs/bunnymark/.

On most devices, it can keep 60 fps at even 50k bunnies, which is impressive. However, iphone 6 plus seems to be an exception. FPS drops to around 30 with just 3k bunnies, and close to zero with 10k. 

Anyone has any clue on what's special with iphone 6 plus?

**Comment by bigtimebuddy** [comment]
Would you mind testing on the latest bunny mark here: https://pixijs.github.io/bunny-mark/

**Comment by xiaobaker** [comment]
@bigtimebuddy this one seems to be heavier than the previous. just tried with default options and I can get up to
* iphone 6: 7k bunnies at 60fps 
* ipad air: 1900 bunnies at 60fps
* iphone 6plus: 3k bunnies at 60fps
* google pixel: 3k bunnies at 60fps

pushing iphone 6plus to 7k bunnies will get about 30fps.

**Comment by xiaobaker** [comment]
@bigtimebuddy ah hold on our "iphone 6" was actually an "iphone 6s" so the A9 v.s. A8 difference might be causing it. we'll look further ourselves and post the findings here.

**Comment by xiaobaker** [comment]
ok tested with a real iphone 6 not 6s this time. here's a summary of our observations:

### with our own game (pixijs 4.5.4, around 500 sprites of only 3 unique textures, each 3MB in size)
* iphone 6: 60fps smooth
* iphone 6s: 60fps smooth
* google pixel: 60fps smooth
* iphone 6 plus: jump between 20-30fps

### with http://www.goodboydigital.com/pixijs/bunnymark/
* iphone 6: 60fps with 30k bunnies
* iphone 6s: 60fps with 50k bunnies
* google pixel: 60fps with 50k bunnies
* iphone 6 plus: 30fps with 3k bunnies   <-- something's very wrong here

### with https://pixijs.github.io/bunny-mark/
* iphone 6: 60fps with 3k bunnies
* iphone 6s: 60fps with 7k bunnies
* iphone 6plus: 3k bunnies at 60fps
* google pixel: 3k bunnies at 60fps

in summary, it seems there's a weird trigger that can cause bad performance on iphone 6 plus, and the latest bunny mark is not hitting that trigger. the previous bunny mark and our game both do.

**Comment by themoonrat** [comment]
What ios version is that 6 plus on?

**Comment by xiaobaker** [comment]
all our ios devices are on 10.3.1

**Comment by siwel** [comment]
I have access to a iPhone 6+ (iOS 9.3.5), I get 60fps with 35k bunnies on http://www.goodboydigital.com/pixijs/bunnymark

**Comment by themoonrat** [comment]
On our 6+, we get 60fps with 30k bunnies on the v3 benchmark, but just under 3k for the v4 benchmark.

**Comment by bigtimebuddy** [comment]
Could you try running https://pixijs.github.io/bunny-mark/ with the `legacy` option enabled?

Wondering if that will improve performance.

![screen shot 2017-08-07 at 10 47 53 am](https://user-images.githubusercontent.com/864393/29031996-f805bd5a-7b5d-11e7-8b64-4272891925be.png)

**Comment by finscn** [comment]
Some issues have talked  that  the performance of v3 is better than v4.
It's so sad.

**Comment by xiaobaker** [comment]
alright we tried a bunch of things, and it seems antialias was the biggest perf hit for our game in this case. enabling antialias on iphone6 and 6plus with ios 10.3.3 is **catastrophic**. my guess is that iphone6 does not have hardware antialiasing but ios 10.3.3 us using some sort of software stuff as polyfill. 

funny thing here @bigtimebuddy - i don't think forceFXAA on latest pixijs is working... i can't find the setting being used anywhere in the codebase. we used this setting in hope of that can give us a good performance on mobile but seems that doesn't work right now. 

for the latest bunny mark test page, I believe it's probably hardware differences between iphone6 and 6s.
